packages/db/src/meta/process/resources.ts

```diff
@@ -83,7 +83,7 @@ export const resourceProcessorsForDefinitions = <C extends Collections>(
         type: "graphql",
         request: gql`
           mutation Load${Resources}(
-            $inputs: [${Resource}Input!]!
+            $inputs: [${Resource}Input]!
```

Comment:
I found out we were wrong about this! The inner `!` doesn't imply "at least one item", it just means "every item will be non-null".

Since this PR broadly says "alright, lots of things inside arrays can be null", result is: remove this constraint.